post.py

```diff
@@ -2,65 +2,97 @@
 from credentials import tokens
-api = twitter.Api(tokens['consumer_key'], tokens['consumer_secret'],
-                  tokens['access_token_key'], tokens['access_token_secret'])
+
+api = twitter.Api(
+      tokens['consumer_key'], tokens['consumer_secret'],
+      tokens['access_token_key'], tokens['access_token_secret']
+)
+
 
 @click.command()
-@click.argument('urlfile',type=click.File('r'))
+@click.argument('urlfile', type=click.File('r'))
 def setUpdate(urlfile):
     try:
         with open('.urlFilePosition', 'r') as position:
             pos = int(position.readline())
-        filelen=len(urlfile.read()) 
-        if pos == filelen:    # Means if pos is at end of file
-            choice = input('The specified file was finished in last iteration. Want to restart it? [y/n]: ')
+        filelen = len(urlfile.read())
+        if pos == filelen:  # Means if pos is at end of file
+            choice = input(
+                'The specified file was finished in last iteration.'
+                'Want to restart it? [y/n]: '
+            )
             if choice.lower() in ['y', 'yes']:
                 pos = 0
             else:
-                click.echo('Please specify the new file as the argument by running the command again.\n')
+                click.echo(
+                    'Please specify the new file as the argument by'
+                    'running the command again.\n'
+                )
                 exit()
         elif pos > filelen:
-            choice = input('The file has been modified to one with shorter lengh. Star from beginning? [y/n]: ')
+            choice = input(
+                'The file has been modified to one with shorter lengh.'
+                'Star from beginning? [y/n]: '
+            )
             if choice.lower() in ['y', 'yes']:
                 pos = 0
             else:
-                click.echo('Please specify the new file as the argument by running the command again.\n')
+                click.echo(
+                    'Please specify the new file as the argument by'
+                    'running the command again.\n'
+                )
                 exit()
         else:
             click.echo('Continuing from the last time.\n')
         urlfile.seek(pos)
     except ValueError:
-            click.echo('The position file is tampered. Starting from beginning.\n')
+            click.echo(
+                'The position file is tampered.'
+                'Starting from beginning.\n'
+            )
     except FileNotFoundError:
-            click.echo('First time usage. No existing file positions found\n')
+            click.echo(
+                'First time usage. No existing file positions found\n'
+            )
     update = ''
     line = urlfile.readline()
-    if not pos:  # Since pos not defined, means first iteration, so skip to place with first link
-        while not line.startswith(('https://','http://')):   # Get to the place with first link
+    if not pos:
+        """
```

Comment:
There's too much line for comment. Isn't it a little hard to find the next indentation level?

Reply:
I hope not because comments are indented at the same level.

Reply:
That's for sure. How about increasing the line length (Keeping in mind the 79 character limit) so that the comment takes up fewer lines? I don't find it comforting that the actual code starts too much down the condition check, given that comment isn't also too long. 
Only a suggestion. If this seems readable to you (Other developers) then it's alright. :)